resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/endpoint/CircuitBreakerHystrixServerSideEvent.java

```diff
@@ -137,5 +117,24 @@ public CircuitBreakerHystrixServerSideEvent(
     private CircuitBreaker getCircuitBreaker(String circuitBreakerName) {
         return circuitBreakerRegistry.circuitBreaker(circuitBreakerName);
     }
+
+    private Flux<ServerSentEvent<String>> getHeartbeatStream() {
```

Comment:
`getHeartbeatStream` to keep connection alive [here](https://stackoverflow.com/questions/48806452/spring-boot-webflux-netty-detect-closed-connection)